spec/support/evm_spec_helper.rb

```diff
@@ -12,6 +12,15 @@
 module EvmSpecHelper
   extend RSpec::Mocks::ExampleMethods
 
+  module EmsFolder
```

Comment:
Also, instead of a helper, can this just be done with a normal FactoryGirl factory?  It would be a lot cleaner if the caller could just do `FactoryGirl.create(:vmware_nested_folders)`